### ISSUE TITLE
focus the tags input without clicking? or autofocus?

### ISSUE THREAD

**Issue by ktmellow** [JohMun/vue-tags-input#65]
This would be good for accessibility.

Something like 
          `  <vue-tags-input
              :focused="booleanVariable">
            </vue-tags-input>`

**Comment by JohMun** [comment]
Yes, it's a good suggestion. I add the label enhancement and implement it in the future.

**Comment by yamin90** [comment]
for now using ref to auto focus
this.$refs.taginput.$refs.newTagInput.focus()

**Comment by talebi** [comment]
had to wrap it inside a nextTick like below:
```
this.$nextTick(function() {
    this.$refs.taginput.$refs.newTagInput.focus();
});
```
